src/System.Linq.Parallel/ref/System.Linq.Parallel.cs

```diff
@@ -16,7 +16,9 @@ public static partial class ParallelEnumerable
     {
         public static TSource Aggregate<TSource>(this System.Linq.ParallelQuery<TSource> source, System.Func<TSource, TSource, TSource> func) { throw null; }
         public static TAccumulate Aggregate<TSource, TAccumulate>(this System.Linq.ParallelQuery<TSource> source, TAccumulate seed, System.Func<TAccumulate, TSource, TAccumulate> func) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.MaybeNullAttribute]
```

Comment:
I assume you went through all of the System.Linq annotations and copied them here?

Reply:
Not exactly, but i did compare and updated accordingly, seems Aggregate functions differ though, will update

Reply:
Turns out this overload version is not exist for System.Linq, I made it nullable because AssociativeAggregationOperator class instance created with throwIfEmpty = false

Reply:
This annotation appears to be incorrect. All code paths I was able to identify directly return the result of a call to `resultSelector`, which would make `TResult` the correct return value.

Reply:
That is right, after loooking back again, when throwIfEmpty is false it is defaulting to `seed` or seedFactory result which will be bound to TAccumulate, also as you said `resultSelector` would make sure. Will update thanks!